### ISSUE TITLE
Link to the xaml element with the binding failure

### ISSUE THREAD

**Issue by linhofen** [spadapet/xaml-binding-tool#23]
Having the name of the view model is really nice. But it would be fun to also have the xaml element where the binding failure happened.

**Comment by spadapet** [comment]
The debug output gives the element type, element name, and property name. I can only parse what's in the debug output string, no other context is given. So there is no way to know for sure what the source XAML element was.

Would you find it useful if double clicking a binding error did the following:
* Initiated a search across all XAML files
* All XAML tag locations would show up in a list if they match:
    * The failed element type
    * The failed element name
    * It contains the property that failed
    * That property value is a {Binding} that matches the failed binding path

That list may show more than one match, but I wonder if you'd find that helpful.

**Comment by linhofen** [comment]
If I understand, instead of manually looking for the xaml or using the search, it would kind of do that step for us. I like it :)

**Comment by spadapet** [comment]
Right, it would help find the exact XAML element given the limited info provided in the debug output. It should be faster than manually looking.

**Comment by charint0** [comment]
I guess it's very difficult to implement, but It would be useful if the search for the xaml element would be limited to the xaml code of the active window at the time the error occurred. This reflects my usual debugging process: 1 Clear the output window. 2 perform some task (open e window etc.) 3. as soon as I see an error in the output: stop any actions and search for the target element in what I think is the current active xaml. This helps when the code base is huge and there are many candidates for the given error.

**Comment by spadapet** [comment]
@charint0 Good idea, thanks!
